src/main/java/no/nav/veilarbaktivitet/domain/Person.java

```diff
@@ -25,6 +25,14 @@ public static NavIdent navIdent(String navIdent) {
         return new NavIdent(navIdent);
     }
 
+    public boolean isExtern() {
+        return this instanceof AktorId || this instanceof Fnr;
+    }
+
+    public InnsenderData toInnsenderData() {
```

Comment:
Jeg ville kalt denne brukerType eller noe sånt som sier hva det er isteden for typen objekt

Reply:
lurte litt på det. InnsenderData er et dårlig navn i utgangspunktet. Jeg renamer metoden i hvert fall